packages/webdriverio/src/utils/getElementObject.ts

```diff
@@ -109,7 +129,7 @@ export const getElements = function getElements(
         ...getWDIOPrototype('element')
     }
 
-    const elements = elemResponse.map((res: ElementReference | Element | Error, i) => {
+    const elements = [elemResponse].flat(1).map((res: ElementReference | Element | Error | ChromedriverErrorResponse, i) => {
```

Comment:
Hi @nextlevelbeard I didn't know this was changed so I love that you created this PR, I got some feedback which I feel might sound harsch but it's very objectively so nothing personal :)

I feel like there is no need for the ChromedriverError class, using the Error class suffices as the ChromedriverError aims to change take the message to either error, name or an empty string. Which should not be part of the class but should be the input (aka the message) and the rest works like it normally would.

However! I do see you use `stacktrace` instead of `stack` and if this is changed in Chromedriver, we shouldn't change how we handle this, instead we should report that `stacktrace` isn't a valid property name for the stacktrace (`stack` is) of an Error https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error instead.

Reply:
Hey @erwinheitzman thanks for having a look at this
As Christian [mentioned](https://github.com/webdriverio/webdriverio/pull/10974#discussion_r1305837268) ChromeDriver is throwing a Error that is according to the W3C spec and we should define it, have a specific interface and inherit the Error class so that it can be understood/handled as any other error, across WDIO code base. At least that's my understanding. We are basically creating a standard error class by mapping the `stacktrace` to `stack`, `error` to `name`, and keeping the `message` property.